public/data/HlobisileLukhele.json

```
@@ -0,0 +1,13 @@
+{
+    "name": "Hlobisile Lukhele",
+    "location": "Durban, South Africa",
+    "bio": "Your Bio should be 20-30 words not more then that",
+    "avatar": "https://github.com/<your-github-username>.png",
```

```suggestion
    "avatar": "https://github.com/HlobisileLukhele.png",
```
Add your username here.